app/services/check_seats_availability.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CheckSeatsAvailability
```

Comment:
Consider that you have two classes with almost the same responsibilities:
1. CheckSeatsAvailability
2. IsEnoughSeats

Both are validators of the ReservationSeat concept. In this case, maybe it would be a good idea to create a separated validator class, like this:

```ruby
module ReservationSeats
  class Validator
    def initialize()
      ...
    end
    
    def valid?
       #logic will be here
    end
  end
end
```

What do you think?